src/Atmos/Model/ref_state.jl

```diff
@@ -95,3 +108,63 @@ function atmos_init_aux!(
     e_pot = gravitational_potential(atmos.orientation, aux)
     aux.ref_state.ρe = ρ * total_energy(e_kin, e_pot, ts)
 end
+
+using ..MPIStateArrays: vars
+using ..DGMethods: init_ode_state
+using ..DGMethods.NumericalFluxes:
+    CentralNumericalFluxFirstOrder,
+    CentralNumericalFluxSecondOrder,
+    CentralNumericalFluxGradient
+
+struct PressureGradientModel <: BalanceLaw end
```

Comment:
Even though it's clear to us who have been following this issue, I think a short docstring describing what this micro balance law is representing is needed.

Reply:
Done.